# math.acos returns -.nan on x86_64, where it returns .nan on other cpu arches.
# upstream confirmed the test should be removed.
--- a/test/parser.fnl	2024-08-24 21:34:54.000000000 -0400
+++ b/test/parser.fnl	2024-09-03 12:28:07.654052734 -0400
@@ -49,8 +49,6 @@
   (t.= "-.inf"
        (fennel.view (fennel.eval "-.inf")))
   (t.= ".nan"
-       (fennel.view (fennel.eval "(math.acos 2)")))
-  (t.= ".nan"
        (fennel.view (fennel.eval ".nan")))
   ;; ensure we consistently treat nan as symbol even on 5.1
   (t.= :not-really (fennel.eval "(let [nan :not-really] nan)"))
